src/components/molecules/AdminInput/AdminInput.scss

```diff
@@ -24,10 +24,9 @@
     &:-webkit-autofill:focus,
     &:-webkit-autofill:active {
       -webkit-text-fill-color: var(--content--over);
-      -webkit-box-shadow: 0 0 0 $input-auto-fill-box-shadow-height
-        $input-auto-fill-box-shadow inset;
-      box-shadow: 0 0 0 $input-auto-fill-box-shadow-height
-        $input-auto-fill-box-shadow inset;
+      -webkit-box-shadow: box-shadow--input-shadow(var(--content--under-80a))
+        inset;
+      box-shadow: box-shadow--input-shadow(var(--content--under-80a)) inset;
```

Comment:
Kind of forgot these would still be duplicates. Don't know why override it with the prefix though, may want to reconsider  at least adding a `@debt` comment to recheck `caniuse.com` in the future and deal with it. Browsers should eventually just use the standard one.

Reply:
removed `--webkit` ones. confirmed that everything is working fine c974b57